ipaserver/install/dsinstance.py

```diff
@@ -1067,7 +1076,9 @@ def add_ca_cert(self, cacert_fname, cacert_name=''):
         self.stop()
 
         dirname = config_dirname(installutils.realm_to_serverid(self.realm))
-        certdb = certs.CertDB(self.realm, nssdir=dirname, subject_base=self.subject_base)
+        certdb = certs.CertDB(
+                self.realm, nssdir=dirname,
+                subject_base=self.subject_base, ca_subject=self.subject)
```

Comment:
Agreed.  (Disagree with first approach though; what if we rename var `certdb`, module `certs` or class `CertDB` and length changes?  You don't want to realign `N` extra lines for all the arguments :)